### PR TITLE
Make level-5.

### PR DESCRIPTION
**Pull request by creator-max**
Added custom collections at this level. Developed class and methods for reading collections from .csv files.

### CODE REVIEW

File: AutoPark/Program.cs [hunk 1]
```diff
@@ -1,37 +1,34 @@
-﻿using System;
-
-namespace AutoPark
+﻿namespace AutoPark
 {
     class Program
     {
         static void Main()
         {
+            const string path = @"C:\Users\ASUS\source\repos\AutoPark\AutoPark\Data\";
```
Comment (PushMeNow) [inline AutoPark/Program.cs:7]: Sorry but I don't have this path on my local comp. Never use absolute path.

**Review by PushMeNow — CHANGES_REQUESTED** [review]

File: AutoPark/Collections.cs [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+
+namespace AutoPark
+{
+    public class Collections
+    {
+        public Collections(string typesPath, string vehiclesPath, string rentsPath)
+        {
+            ListVehicleType = LoadTypes(typesPath);
+            ListVehicle = LoadVehicles(vehiclesPath);
+            LoadRents(rentsPath);
```
[the remark this answers is not in the document]
Comment (creator-max) [reply AutoPark/Collections.cs]: Thanks, fixed

File: AutoPark/Collections.cs [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+        }
+
+        public List<VehicleType> ListVehicleType { get; }
+        public List<Vehicle> ListVehicle { get; }
```
[the remark this answers is not in the document]
Comment (creator-max) [reply AutoPark/Collections.cs]: fixed

File: AutoPark/Program.cs (hunk 1 above)
Comment (creator-max) [reply AutoPark/Program.cs]: fixed

File: AutoPark/Program.cs [hunk 4]
```diff
@@ -1,37 +1,37 @@
 ﻿using System;
+using System.IO;
 
 namespace AutoPark
 {
     class Program
     {
         static void Main()
         {
+            string path = Path.GetFullPath(Path.Combine(AppContext.BaseDirectory, @"..\..\..\Data\"));
```
Comment (PushMeNow) [inline AutoPark/Program.cs:10]: Again bad solution. These files must be copied on compile and you have to get them from base directory.

**Review by PushMeNow — CHANGES_REQUESTED** [review]

File: AutoPark/Program.cs (hunk 4 above)
Comment (creator-max) [reply AutoPark/Program.cs]: Added

**Review by PushMeNow — APPROVED** [review]

**Review by kuzmiich — APPROVED** [review]
good job

**Pull request merged**